src/plans/D3MCompoundPlan.sol

```diff
@@ -0,0 +1,199 @@
+// SPDX-FileCopyrightText: © 2021-2022 Dai Foundation <www.daifoundation.org>
```

Comment:
Following recent guidance clarification from Dai Foundation on discord, it emerged that only the year of file creation would need to be noted.
```suggestion
// SPDX-FileCopyrightText: © 2021 Dai Foundation <www.daifoundation.org>
```

Reply:
For `Hub` and  `Aave` is arguable as the logic itself comes from the older version which was done last year.

Reply:
There are still the old copyright comments though, should those be removed in favour of the new format?

Reply:
Following latest guidance updates, the old copyright format should be replaced with the new one.

Reply:
Left suggestions below.